src/VisualStudio/TestUtilities/VisualStudioInstanceFactory.cs

```diff
@@ -79,7 +78,7 @@ private void StartNewInstance()
         private static Process StartNewVisualStudioProcess()
         {
             // TODO: This might not be needed anymore as I don't believe we do things which risk corrupting the MEF cache. However,
-            // it is still useful to do in case some other action corruped the MEF cache as we don't have to restart the host
+            // it is still useful to do in case some other action corrupted the MEF cache as we don't have to restart the host
             Process.Start(VsExeFile, $"/clearcache {VsLaunchArgs}").WaitForExit();
```

Comment:
I've now deleted this equivalent code in our old system. We should delete this here too.